Automattic/vip-go-mu-plugins
# Update Jetpack to 9.4.1

## Description

Update `jetpack-9.4` subtree to the latest Jetpack version 9.4.1

## Changelog Description

### Plugin Updated: Jetpack 9.4.1

Web have upgraded Jetpack 9.4 subtree to Jetpack 9.4.1, the latest patch release.

Not a lot of changes, just [a few bugfixes and improvements](https://github.com/Automattic/jetpack-production/releases/tag/9.4.1).

## Checklist


## Steps to Test

1. Check out PR.
1. Go to the Jetpack dashboard in wp-admin
1. Ensure the active version is 9.4.1
1. Ensure everything keeps working normally
